__tests__/games.test.js

```diff
@@ -98,4 +98,78 @@ afterAll(() => {
           
       });
       
+  });
+
+  describe.only('PATCH /api/reviews/review_id', () => {
+      test('202: responds with a review with it\'s votes counter altered by the correct amount', () => {
+          const inc_votes = { inc_votes: -5 }
+
+          return request(app)
+          .patch("/api/reviews/2")
+          .send(inc_votes)
+          .expect(202)
+          .then((results) => {
+            const review = results.body.review
+            
+            expect(review.length).toBe(1)
+
```

Comment:
As we are sending back a single review I would take the review object out of the array before sending as a response. It's convention to put multiples in an array but send a single as an object